### ISSUE TITLE
0.0.8 only showing Zappi

### ISSUE THREAD

**Issue by cconnoruk** [CJNE/ha-myenergi#18]
0.0.7 showed the Zappi, Harvi, Hub but only the Zappi is now showing in 0.0.8

I have just removed a custom setup and re-installed and setup via HACS.

**Comment by CJNE** [comment]
It sounds like something goes wrong, have you check the log for any errors?

**Comment by cconnoruk** [comment]
I was about to say nothing in there but ... 

```
Logger: homeassistant.components.sensor
Source: custom_components/myenergi/sensor.py:131
Integration: Sensor (documentation, issues)
First occurred: 1:08:06 pm (2 occurrences)
Last logged: 1:08:17 pm

Error while setting up myenergi platform for sensor
Traceback (most recent call last):
  File "/snap/home-assistant-snap/365/lib/python3.8/site-packages/homeassistant/helpers/entity_platform.py", line 249, in _async_setup_platform
    await asyncio.shield(task)
  File "/var/snap/home-assistant-snap/365/custom_components/myenergi/sensor.py", line 131, in async_setup_entry
    totals = coordinator.client.get_power_totals()
  File "/root/snap/home-assistant-snap/365/.local/lib/python3.8/site-packages/pymyenergi/client.py", line 88, in get_power_totals
    if device.ct3.is_assigned:
AttributeError: 'Eddi' object has no attribute 'ct3'
```

**Comment by CJNE** [comment]
Ah! It's fixed in 0.0.10 which was just released :)
